src/decisionengine_modules/glideinwms/glide_frontend_element.py

```diff
@@ -1855,7 +1859,7 @@ def count_match_fom_bff(self, job_types, job_type, entries):
 
         total = job_types[job_type]['abs']
         # TODO: MMDB should be debug
-        # logger.info('\n'.join(dbg_info))
+        # self.logger.info('\n'.join(dbg_info))
```

Comment:
Commented out line?

Reply:
yes, I thought to change it anyway in case it gets uncommented in the future. Very unlikely since this module will need to be all rewritten, but still just in case.